lib/oxidized/model/hpvc.rb

```diff
@@ -0,0 +1,22 @@
+class Hpvc < Oxidized::Model
+  # HPE Moonshot Switch / HP Virtual Connect Linux
+  # sometimes the prompt might have a leading nul or trailing ASCII Bell (^G)
+  prompt /^\0*(\([\w.-]+\)).?[>#]$/
+  comment '# '
+
+  cfg :telnet, :ssh do
+    post_login 'enable'
+    post_login 'terminal length 0'
+
+    pre_logout "exit\nquit\n"
+  end
+
+  cmd 'show version' do |cfg|
+    comment cfg
+  end
+
+  cmd 'show running-config' do |cfg|
+    cfg.gsub! /^\!System Up Time .*$/, '\\1 <removed>'
+    cfg.gsub /^\s+/, ''
```

Comment:
Oxidized normally prefers to store configurations as-returned. If the goal is instead to compact the file, it may be unnecessary.

Is the goal here instead to remove a prefix or suffix empty line? Consider the .cut_head, .cut_tail, or .cut_both methods. 

For clarity, it may be useful to return 'cfg' explicitly as a last statement.